python/paddle/fluid/async_executor.py

```diff
@@ -77,6 +78,12 @@ class AsyncExecutor(object):
     """
 
     def __init__(self, place=None, run_mode=""):
+        """
```

Comment:
any comments, please preview first, thanks!
refer to [here](http://wiki.baidu.com/pages/viewpage.action?pageId=705892848)